.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: node_js
 
-before_script:
- - sh -e /etc/init.d/xvfb start
-
-install: ant
-
 env:
- - DISPLAY=':99.0'
+  global:
+    - DISPLAY=':99.0'
+  matrix:
+    - MOZDOWNLOADPARAMS='-t daily'
+    - MOZDOWNLOADPARAMS='-t daily --branch=mozilla-aurora'
+    - MOZDOWNLOADPARAMS='-t release -v latest-beta'
+    - MOZDOWNLOADPARAMS='-t release -v latest'
+    - MOZDOWNLOADPARAMS='-t release -v latest-esr'
 
-script: ant test
+before_install:
+  - sudo apt-get update -qq
 
-notifications:
-  irc:
-    channels:
-      - "irc.mozilla.org#automation"
-    on_success: always
-    on_failure: always
-    use_notice: true
+install:
+  - sudo apt-get install -y ant python-pip
+  - sudo pip install mozdownload
+
+before_script:
+  - sh -e /etc/init.d/xvfb start
+  - mozdownload $MOZDOWNLOADPARAMS
+  - tar -xf *firefox*.tar.bz2
```

Comment:
Yes it is cleared. See [Travis CI doc][1]!

Addon SDK prints the version in the build, you could check that too! It isn't the same.
```
[exec] Running tests on Firefox 37.0a2/Gecko 37.0a2 ({ec8030f7-c20a-464f-9b0e-13a3a9e97384}) under linux/x86_64.
```

[1]: http://docs.travis-ci.com/user/build-configuration/#Travis-CI-Preserves-No-State-Between-Builds "Travis CI Preserves No State Between Builds"